Ficek_QuantiApp/Ficek_QuantiApp/Domain/Tests/RocketConvertersTests/RocketConverter+sadTests.swift

```diff
@@ -10,95 +10,95 @@ import ModelConvertible
 import Networking
 
 final class RocketRonverterSadTests: XCTestCase {
```

Comment:
Testy musi 1:1 odpovidat produkcni adresarovy strukture. Takze v RocketClientTests > Converters.

Reply:
To neodpovida... Prvni je `{TargetName}Tests`, protoze to tak vyzaduje SPM. Pak uz ty nazvy podadresaru musi odpovidat presne produkcnimu kodu. Tady `Converters`.